### PR TITLE
Add metrics about grpc async client and server connections

### PR DESCRIPTION
**Pull request by xzhangxian1008**
### What problem does this PR solve?

Issue Number: ref #6225 

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

You can see these two metric in tiflash_summary/Threads_CPU [here](http://172.16.4.99:18501/)

image

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by LittleFall** [comment]
can you add some metric screenshots in pr description? we may check it in the future.

**Review by LittleFall — COMMENTED** [review]
add a "limit" line like other metrics in this section?

File: metrics/grafana/tiflash_summary.json [hunk 1]
```diff
@@ -1,40 +1,4 @@
...
-      "name": "Heatmap",
-      "version": ""
-    },
-    {
-      "type": "datasource",
-      "id": "prometheus",
-      "name": "Prometheus",
-      "version": "1.0.0"
-    }
-  ],
```
Comment (windtalker) [inline metrics/grafana/tiflash_summary.json:37]: Why delete these?

**Comment by xzhangxian1008** [comment]
> add a "limit" line like other metrics in this section?

done

File: metrics/grafana/tiflash_summary.json (hunk 1 above)
Comment (xzhangxian1008) [reply metrics/grafana/tiflash_summary.json]: I have revoked it

**Review by LittleFall — APPROVED** [review]

**Review by windtalker — APPROVED** [review]
LGTM

**Pull request merged**